dynawo/sources/Solvers/SolverIDA/test/TestBasics.cpp

```diff
@@ -261,16 +255,14 @@ TEST(SimulationTest, testSolverIDATestAlpha) {
   // The event in the model is written as if (x <= 0) which means that the root is detected just after t = 2 (2 + epsilon).
   // IDA will thus stop just after t = 2 and is reinitialized (thus the next time step will be t = 3)
   ASSERT_NO_THROW(solver->reinit());
-  ASSERT_EQ(solver->getPreviousReinit(), Algebraic);
+  ASSERT_EQ(solver->getPreviousReinit(), AlgebraicWithJUpdate);
```

Comment:
Why does the evalMode status change? 
This should be somehow related to the discrete call treatment

Reply:
This was caused by the issue with mode change in calculated variables. Solved.